src/psyclone/transformations.py

```diff
@@ -1076,22 +1121,30 @@ def apply(self, node, collapse=None, independent=True, sequential=False):
 
         :param node: the supplied node to which we will apply the \
                      Loop transformation.
-        :type node: :py:class:`psyclone.psyGen.Loop`.
-        :param int collapse: number of loops to collapse into single \
-                             iteration space or None.
-        :param bool independent: whether to add the "independent" clause to \
-                                 the directive (not strictly necessary within \
-                                 PARALLEL regions).
-        :returns: (:py:class:`psyclone.psyGen.GOInvokeSchedule`, \
-                  :py:class:`psyclone.undoredo.Memento`)
+        :type node: :py:class:`psyclone.psyGen.Loop`
+        :param options: a dictionary with options for transformations.\
+                        This transform supports:\
+                        - "collapse" (int): number of nested loops to\
```

Comment:
Please use the same form as in other places for documenting the options, e.g.:

:param int options["collapse"]: number of ...

Reply:
Done.